pluto-plugins/plugins/network/core/lib-no-op/src/main/java/com/pluto/plugins/network/intercept/NetworkData.kt

```diff
@@ -11,6 +11,7 @@ class NetworkData {
     )
 
     data class Response(
+        val request: Request,
```

Comment:
this is an anti-pattern,
Request & Response are mutually exclusive objects.

can understand what you are trying to do here, but lets figure out a better way, than corrupting the Response class.

Reply:
fixed

Reply:
yes thats right, 
we have ApiCallData for that purpose only.